### PR TITLE
SG-21570 beyond you first project

### PR DESCRIPTION
**Pull request by zamu5**

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/32755eb4338c4995ef187c164770bd25708155cb/index.html)

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 1]
```diff
@@ -5,4 +5,212 @@ pagename: beyond-your-first-project
...
+  
+      + [Key Configuration Files](#key-configuration-files)  
+  
+      + [Core API platform](#core-api-platform)  
+  
+      + [Further reading](#further-reading)  
+  
+  * [The pipeline Toolkit Community](#the-pipeline-toolkit-community)  
+  
+  * [Our Support Team](#our-support-team)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:43]: Let's remove the table of contents entirely. What you have here actually looks pretty good, but I don't think the document is long enough to really need one explicitly built into the document like this.

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 2]
```diff
@@ -5,4 +5,212 @@ pagename: beyond-your-first-project
...
+  
+  * [The pipeline Toolkit Community](#the-pipeline-toolkit-community)  
+  
+  * [Our Support Team](#our-support-team)  
+  
+# Welcome to Toolkit  
+  
+Welcome to Toolkit! If you are reading this, it probably means that you have managed to successfully install your first {% include product %} Pipeline Toolkit Project using the {% include product %} Desktop.  
+  
+![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/55/no_version_0513_171143/beyond_your_first_project/project_ready.png)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:49]: Already mentioned this to you elsewhere, but let's get the images into the repo and reference them directly. I'd prefer we stay away from external links like this where possible.

**Review by thebeeland — CHANGES_REQUESTED** [review]
Couple things to sort out, but in general I think it looks pretty good. I'll give it a thorough read once you've made the requested changes just to check on whether any content needs to be updated, as some of this document has been around a long time and might benefit from new info. I'll handle that last part, though, once you polish up those last couple things I mentioned below.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/13ef8b73e06ac67d37f489174d506648c16cc5b9/index.html)

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 3]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+  
+Open this file and make the necessary changes to the paths. Once you have saved the file, you need to leave the project inside of {% include product %} desktop and then click back into it. (but no need to restart the entire application).  
+  
+**Further Reading**  
+  
+For more information about applications, check out the following topics:  
+   
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/apps/tk-multi-launchapp/v0.10.2_1029_133142/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219032968) [> The toolkit application launcher](https://support.shotgunsoftware.com/hc/en-us/articles/219032968)  
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/apps/tk-multi-launchapp/v0.10.2_1029_133142/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219032968#Use%20Command%20Line%20Arguments%20at%20Launch) [> Passing Command line Arguments](https://support.shotgunsoftware.com/hc/en-us/articles/219032968#Use%20Command%20Line%20Arguments%20at%20Launch)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:49]:
```suggestion
- [The Toolkit Application Launcher](https://support.shotgunsoftware.com/hc/en-us/articles/219032968)  
- [Passing Commandline Arguments](https://support.shotgunsoftware.com/hc/en-us/articles/219032968#Use%20Command%20Line%20Arguments%20at%20Launch)    
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 4]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+  
+This next section is all about what to do next -- the process of starting to take that default configuration and adjust it to work more like the rest of your studio pipeline. Toolkit is flexible and highly configurable, and we have lots of documentation. But before you get started, to see it all in action, we recommend spending a couple of minutes checking out our various walkthrough videos. These show the {% include product %} Pipeline Toolkit in action, how it works inside applications such as Maya and Nuke. It also goes through basic concepts such as publishing, version control, loading etc.  
+  
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/17/no_version_0530_193825/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219040678) [> {% include product %} Pipeline Toolkit video collection](https://support.shotgunsoftware.com/hc/en-us/articles/219040678)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:112]:
```suggestion
[{% include product %} Toolkit Video Collection](https://support.shotgunsoftware.com/hc/en-us/articles/219040678)  
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 5]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+  
+If you navigate to your project configuration, you can see a  `tank` and a  `tank.bat` command in the root of the configuration. Running these commands without any options will give you a list of all the commands that are supported in your current configuration, including the following useful commands:  
+  
+- `tank shell` - Start an interactive python shell with tk api access  
+- `tank core` - Check if there are any core API updates available for this project  
+- `tank updates` - Check if any of the apps or engines in this configuration has got any updates available  
+  
+For more details on what you can do with the  `tank` command, please see the in-depth technical documentation:  
+  
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/11/no_version_0530_204710/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033178) [> How to administer Toolkit.](https://support.shotgunsoftware.com/hc/en-us/articles/219033178)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:142]:
```suggestion
[How to Administer Toolkit](https://support.shotgunsoftware.com/hc/en-us/articles/219033178)  
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 6]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+  
+Hand in hand with this goes the Toolkit  _template system_ which makes it easy to define the various paths to files that you can configure; your publishes, work files, renders etc. This is stored in the  `templates.yml` file above.  
+  
+Together, these two parts of the project configuration makes it possible to adjust the various Apps that toolkit use to write out data to locations on disk which make sense are are understood by your existing pipeline. Read more about this in our advanced documentation:  
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/11/no_version_0530_204710/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Creating%20folders%20on%20disk%20with%20Sgtk) [> Folder Configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Creating%20folders%20on%20disk%20with%20Sgtk)  
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/11/no_version_0530_204710/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Configuring%20Templates) [> File system templates](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Configuring%20Templates)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:158]:
```suggestion
Together, these two parts of the project configuration makes it possible to adjust the various Apps that toolkit use to write out data to locations on disk which make sense are are understood by your existing pipeline.

Read more about this in our advanced documentation:  

- [Folder Configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Creating%20folders%20on%20disk%20with%20Sgtk)  
- [Filesystem Templates](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Configuring%20Templates)  
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 7]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/11/no_version_0530_204710/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Configuring%20Templates) [> File system templates](https://support.shotgunsoftware.com/hc/en-us/articles/219033178#Configuring%20Templates)  
+  
+A toolkit configuration is essentially made up of a collection of configured  **apps and engines**. This configuration is located in the  `env` folder. If the file system configuration files discussed above define  _where_ resources should be located on disk, the environment configuration with its apps and engines define  _what_ the pipeline is supposed to do.  
+
+[![](https://sg-toolkit.s3.amazonaws.com/tank/zendesk_appstore/tank_logo_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219039798) [> The official Toolkit App and Engine listing.](https://support.shotgunsoftware.com/hc/en-us/articles/219039798)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:162]:
```suggestion
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 8]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
...
+  
+The configuration, apps and engines are all running on top of the Toolkit Core platform. For new projects, this is also stored inside the  `install` folder. Essentially, a project configuration is fully self contained - all the necessary pieces required to run toolkit are in a single place. This also means that each project is independent and updating one project will not break another.  
+  
+Tech Notes: Using a shared Toolkit Core (Click to expand)  
+  
+### Further reading  
+  
+We also have a more technical document that goes through the high level concepts in the {% include product %} Pipeline Toolkit and explains 'bigger picture' things. Once you have a good grasp of what Toolkit does out of the box, we recommend that you move on to this document to get a deeper undestanding of how Toolkit could be adjusted to suit your particular studio needs.  
+  
+[![](https://sg-toolkit.s3.amazonaws.com/tank/docs/docs/10/no_version_0330_043251/icon_256.png)](https://support.shotgunsoftware.com/hc/en-us/articles/219040648) [> An introduction to the high level concepts in the Shotgun Pipeline Toolkit.](https://support.shotgunsoftware.com/hc/en-us/articles/219040648)
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:176]:
```suggestion
[An introduction to the high level concepts in the {% include product %} Toolkit](https://support.shotgunsoftware.com/hc/en-us/articles/219040648)  
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 9]
```diff
@@ -5,4 +5,182 @@ pagename: beyond-your-first-project
 lang: en
 ---
 
-# Beyond Your First Project
+  
+# Beyond your first project  
+  
+Here, we explain where to go once you have got your first project up and running using the {% include product %} Desktop. It covers useful common questions and topics and lists useful documentation resources.    
+  
+_Please note that this document describes functionality only available if you have taken control over a Toolkit configuration. For details, see the  [{% include product %} Integrations User Guide](https://support.shotgunsoftware.com/hc/en-us/articles/115000068574-Integrations-User-Guide#Getting%20Started%20with%20Shotgun%20Desktop)._
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:13]:
```suggestion
```

**Review by thebeeland — CHANGES_REQUESTED** [review]
A little cleanup of some images I don't think we need that were used as thumbnails for links in the document. I'm just removing them and will handle committing that now.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/762d458759495c7f140581de3ad8009dd818ba51/index.html)

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 10]
```diff
@@ -5,4 +5,180 @@ pagename: beyond-your-first-project
...
+  
+Tech Notes: Using a shared Toolkit Core (Click to expand)  
+  
+### Further reading  
+  
+We also have a more technical document that goes through the high level concepts in the {% include product %} Pipeline Toolkit and explains 'bigger picture' things. Once you have a good grasp of what Toolkit does out of the box, we recommend that you move on to this document to get a deeper undestanding of how Toolkit could be adjusted to suit your particular studio needs.  
+  
+[An introduction to the high level concepts in the {% include product %} Toolkit](https://support.shotgunsoftware.com/hc/en-us/articles/219040648)  
+  
+## The pipeline Toolkit Community
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:176]:
```suggestion
## The Toolkit Community  
```

File: docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md [hunk 11]
```diff
@@ -5,4 +5,180 @@ pagename: beyond-your-first-project
...
+  
+## The pipeline Toolkit Community  
+  
+A part of Toolkit is its community of pipeline engineers and TDs! We are on a mission to create a vibrant, code sharing community where we all can help evolve Toolkit together to become a powerful and flexible pipeline environment.  
+  
+If you have any questions, or want to read through existing posts and conversations, please visit our  [public forums section](https://support.shotgunsoftware.com/hc/en-us/community/topics/200682428-Pipeline-Toolkit-Common-Questions-and-Answers).  
+  
+## Our Support Team  
+  
+Lastly, if you have any questions, ideas or concerns, never hesitate to contact us directly! We try be be practical and hands on with support and if you for example have found a bug, we can often push a bug fix update quickly to you via our app store. Just drop us a line on  `support@shotgunsoftware.com` and we'll be happy to help!
```
Comment (thebeeland) [inline docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md:184]:
```suggestion
```

**Review by thebeeland — APPROVED** [review]
Looks good! I'm going to commit the last couple things at the end of the doc and then will merge. These last two suggestions are just one modernization and removing a bit about our support email address at the end.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/e6540badc3cdf4561f779919d58f2a071d124370/index.html)

**Pull request merged**